packages/flutter/test/material/paginated_data_table_test.dart

```diff
@@ -152,7 +152,7 @@ void main() {
     ));
 
     // the column overflows because we're forcing it to 600 pixels high
-    expect(tester.takeException(), contains('A vertical RenderFlex overflowed by'));
+    expect(tester.takeException(), contains('A RenderFlex overflowed by'));
```

Comment:
:-(

Reply:
Yeah, I know...  Since the overflow code is now factored out, including specific info is harder.  I could give the overflow mixin a way to designate the "description" of the widget it's attached to, but that seemed more complicated.